postgres/datadog_checks/postgres/data/conf.yaml.example

```diff
@@ -182,10 +183,10 @@ instances:
 ## port / path / channel_path - required - Set port if type is tcp or udp.
 ##                                         Set path if type is file.
 ##                                         Set channel_path if type is windows_event.
-## source  - required - Attribute that defines which Integration sent the logs.
+## source  - required - Attribute that defines which Integration sent the logs
```

Comment:
```suggestion
## source  - required - Attribute that defines which Integration sent the logs.
```

Reply:
Strange, that is what `ddev` changed:
```
ddev --version
ddev, version 3.11.0
```
Thanks, will update manually!

Reply:
Looks like there was a PR recently to fix the logs typos in the template that hasn't been released yet. Sorry about that!